docs/source/hardware_and_software_configuration/configuring/configuring_dual_hubs/configuring-dual-hubs.rst

```diff
@@ -54,22 +39,123 @@ need the following items:
      - .. figure:: images/TamiyaAdapter.jpg
 
    * - FIRST-approved 12V Battery (such as Tetrix W39057). For a list of
-       FIRST-approved 12V batteries, refer to the current Game Manual
-       Part 1, rule <RE03>.
+       FIRST-approved 12V batteries, refer to the current Competition Manual,
```

Comment:
Rephrase to not point to s specific rule number. Rule numbers may shift year to year.  

Suggest something like:

> Check the Robot Construction rules in the [Competition Manual](https://ftc-resources.firstinspires.org/file/ftc/game/manual) for details.